pkgs/by-name/be/berry/package.nix

```diff
@@ -51,6 +55,8 @@ stdenv.mkDerivation (finalAttrs: {
     patchShebangs configure
   '';
 
+  env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.cc.isClang "-Wno-error=implicit-function-declaration";
+
```

Comment:
Can we try the following?

```
env.NIX_CFLAGS_COMPILE = lib.optionalString stdenv.hostPlatform.isDarwin "-D_C99_SOURCE";
```

Because what's happening here seems to be a Darwin specific problem; `vsnprintf` isn't defined unless `_C99_SOURCE` is: https://github.com/apple-open-source-mirror/Libc/blob/master/include/stdio.h#L390

Reply:
Looks like that works, too.